src/script/auth/page/ConversationJoin.tsx

```diff
@@ -124,10 +124,12 @@ const ConversationJoin = ({
     }
   }, []);
 
-  const routeToApp = () => {
+  const routeToApp = (conversation?: string, domain?: string) => {
     const redirectLocation = isPwaEnabled
       ? UrlUtil.pathWithParams(EXTERNAL_ROUTE.PWA_LOGIN, {[QUERY_KEY.IMMEDIATE_LOGIN]: 'true'})
-      : UrlUtil.pathWithParams(EXTERNAL_ROUTE.WEBAPP);
+      : `${UrlUtil.pathWithParams(EXTERNAL_ROUTE.WEBAPP)}${
+          conversation && `#/conversation/${conversation}${domain ? `/${domain}` : ''}`
+        }`;
```

Comment:
This one still can have undefined after `UrlUtil.pathWithParams(EXTERNAL_ROUTE.WEBAPP)`, maybe add default parameter conversation as empty string :) Also you can pass for domain empty string and then you can use `#/conversation/${conversation}${domain && `/${domain}`}`
:)

Reply:
yep i forgot this one was stacked